scripts/ci/dump_diagnostics.sh

```diff
@@ -24,6 +27,9 @@ for pod_name in $(kubectl get pod -n "${namespace}" --output=jsonpath={.items..m
   echo "Writing Pod yaml for pod ${pod_name}"
   kubectl get pod "${pod_name}" -o yaml > "diagnostics/${pod_name}.yaml"
 
+  # dump agent output
+  kubectl cp "${pod_name}":/var/log/mongodb-mms-automation -c mongodb-agent diagnostics/"${pod_name}-mongodb-automation"/
```

Comment:
As seen here: https://github.com/mongodb/mongodb-kubernetes-operator/actions/runs/3883244986

tree output will be:
```
❯ tree | rg automation
├── mdb-tls-0-mongodb-automation
├── mdb-tls-1-mongodb-automation
├── mdb-tls-2-mongodb-automation
```